tensorboard/components/tf_tensorboard/tf-tensorboard.html

```diff
@@ -664,7 +679,10 @@
             return;
           }
           const activePlugins = result.value;
-          this._activeDashboards = this._dashboards.filter(d => activePlugins[d]);
+          this._activeDashboards = _.map(
```

Comment:
Do you think that this is easier to read as `this._dashboardInformationObjects.map(d => d.plugin).filter(p => activePlugins[p])`?

Reply:
Yes! Producing a list of strings and then filtering does seem clearer - the filtering logic only needs to observe the string value. Done.